packages/back-end/src/models/FeatureRevisionModel.ts

```diff
@@ -302,12 +334,98 @@ export async function markRevisionAsPublished(
   );
 }
 
+export async function markRevisionAsReviewRequested(
+  revision: FeatureRevisionInterface,
+  user: EventAuditUser,
+  comment?: string
+) {
+  const action = "Review Requested";
+
+  const log: RevisionLog = {
+    action,
+    subject: "",
+    timestamp: new Date(),
+    user,
+    value: JSON.stringify(comment ? { comment } : {}),
+  };
+  migrateToPendingReview(revision.organization, revision.featureId);
+  await FeatureRevisionModel.updateOne(
+    {
+      organization: revision.organization,
+      featureId: revision.featureId,
+      version: revision.version,
+    },
+    {
+      $set: {
+        status: "pending-review",
+        publishedBy: user,
+        datePublished: new Date(),
```

Comment:
Should these publish settings be updated here if a review is just being requested?  Nothing is being published yet.